tests/rmq_listen.cc

```diff
@@ -0,0 +1,26 @@
+#include <silicon/api.hh>
+#include <silicon/backends/rabbitmq.hh>
+
+iod_define_symbol(test)
+iod_define_symbol(test1)
+iod_define_symbol(test2)
+iod_define_symbol(message)
+
+auto hello_api = sl::http_api(
+		s::_test / s::_test1 * sl::get_parameters(s::_id = int()) = [] (auto				params)
+		{
+			std::cout << "coucou test/test1 " << params.id << std::endl;
+			return D(s::_message = "test/test1");
+		},
+		s::_test / s::_test2 * sl::get_parameters(s::_str = std::string()) = [] (auto		params)
+		{
+			std::cout << "coucou test/test2 " << params.str << std::endl;
+			return D(s::_message = "test/test1");
+		}
+);
+
+int main(int argc, char* argv[])
+{
+	auto ctx = sl::rmq::serve(hello_api, atoi(argv[2]), s::_hostname		= std::string(argv[1]),
```

Comment:
Once we a RQM C++ procucer it will be nice to test if rmq::serve has no bugs.
I did it for the HTTP backend here: https://github.com/matt-42/silicon/blob/master/tests/backend_testsuite.hh